subprojects/glib.wrap

```diff
@@ -1,9 +1,6 @@
-[wrap-file]
-directory = glib-2.68.1
-
-source_url = https://download.gnome.org/sources/glib/2.68/glib-2.68.1.tar.xz
-source_filename = glib-2.68.1.tar.xz
-source_hash = 241654b96bd36b88aaa12814efc4843b578e55d47440103727959ac346944333
+[wrap-git]
+url = https://gitlab.gnome.org/gnome/glib.git
+revision = main
```

Comment:
Is this meant to be part of this PR?

Reply:
Oops, sorry, leftover from some unrelated tests I did. `git add -p` followed by `git commit -a`....

Dropped that now.